src/tests/ftest/util/telemetry_test_base.py

```diff
@@ -84,3 +84,181 @@ def verify_telemetry_list(self, with_pools=False):
             self.fail("\n".join(errors))
 
         self.log.info("Test PASSED")
+
+    def get_min_max_mean_stddev(self, suffix, num_targets):
+        """Get four lists of min, max, mean, stddev.
+
+        Sample get_pool_metrics output.
+        {
+            'engine_io_ops_dkey_enum_latency_min': {
+                'wolf-31': {
+                    '0': {
+                        '0': 6,
+                        '1': 13,
+                        '2': 7,
+                        '3': 6,
+                        '4': 6,
+                        '5': 8,
+                        '6': 10,
+                        '7': 12
+                    }
+                },
+                'wolf-32': {
+                    '1': {
+                        '0': 10,
+                        '1': 9,
+                        '2': 7,
+                        '3': 10,
+                        '4': 18,
+                        '5': 9,
+                        '6': 6,
+                        '7': 11
+                    }
+                }
+            },
+            ...
+
+        Args:
+            suffix (str): Metrics suffix for the metric that has min, max,
+                mean, or stddev at the end.
+            num_targets (int): Number of targets.
+
+        Returns:
+            dict: min, max, mean, stddev from each target in a dict.
+                {
+                    "min": [0, 1, 2, 3, 4, 5, 6, 7...],
+                    "max": [0, 1, 2, 3, 4, 5, 6, 7...],
+                    "mean": [0, 1, 2, 3, 4, 5, 6, 7...],
+                    "stddev": [0, 1, 2, 3, 4, 5, 6, 7...]
+                }
+                Note that all the stats should have the same rank order for the
+                verification to work. Not using the sample input values.
+
+        """
+        output = {}
+
+        for stats_value in ["min", "max", "mean", "stddev"]:
+            specific_metrics = [suffix + stats_value]
+            pool_out = self.telemetry.get_pool_metrics(
+                specific_metrics=specific_metrics)
+            self.log.info("pool_out = %s", pool_out)
+
+            values = [0 for _ in range(num_targets)]
```

Comment:
This also works, but not necessary to change
```suggestion
            values = [0] * num_targets
```

Reply:
Done. Thanks for the tip.